README.md

```diff
@@ -34,21 +34,24 @@ libraries and such might not want.
 
 ### Table of Contents <!-- omit in toc -->
 
-- [About](#about)
-- [Features](#features)
-- [Installing](#installing)
-- [Example](#example)
-- [Using](#using)
-  - [... in libraries](#-in-libraries)
-  - [... in application code](#-in-application-code)
-  - [... in `main()`](#-in-main)
-  - [... diagnostic code URLs](#-diagnostic-code-urls)
-  - [... snippets](#-snippets)
-  - [... multiple related errors](#-multiple-related-errors)
-  - [... delayed source code](#-delayed-source-code)
-  - [... handler options](#-handler-options)
-- [Acknowledgements](#acknowledgements)
-- [License](#license)
+- [`miette`](#miette)
```

Comment:
Whoops. My Markdown VSCode extension automatially updated this TOC. I'll disable that and recommit.